lib/e2e/src/system.rs

```diff
@@ -30,36 +33,11 @@ pub type Wallet = FillProvider<
     Ethereum,
 >;
 
-/// Convenience type alias that represents an alloy provider.
-pub type Provider = FillProvider<
```

Comment:
@bidzyyys I haven't found any place where this Provider was used. Just drop it as I see